lib/assessment/label.py

```diff
@@ -115,9 +115,8 @@ def bedrock_anthropic_label_student_work(self, prompt, rubric, student_code, stu
         response = bedrock.invoke_model(body=body, modelId=bedrock_model, accept=accept, contentType=content_type)
 
         response_body = json.loads(response.get('body').read())
-        generation = response_body.get('completion')
 
-        data = self.get_response_data_if_valid(generation, rubric, student_id, response_type='json')
+        data = self.get_response_data_if_valid(response_body, rubric, student_id, response_type='json')
```

Comment:
Note: I added the changes needed to process Claude into this PR because I still haven't fixed my OpenAI connection. I can remove these if we would rather have them as a separate update.